lua/rocks-config/init.lua

```diff
@@ -94,7 +94,29 @@ function rocks_config.setup(user_configuration)
         end
     end
 
+    local exclude = {}
+
+    if type(config.plugins and config.plugins.bundles) == "table" then
+        for bundle_name, plugins in pairs(config.plugins.bundles) do
```

Comment:
question: would it make sense to check if each plugin in a bundle is actually installed (and maybe warn if that's not the case)?

Reply:
I'm unsure what the performance implications of this would be - maybe doing this in the healthcheck would be a better decision? Polling luarocks for the installed plugin list sounds like not the best idea on startup :sweat_smile:

Reply:
Oh of course, that I can get behind :D